[2022-12-29T18:16:20.780+0100] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: spark_airflow_project.python_job manual__2022-12-29T17:16:13.051785+00:00 [queued]>
[2022-12-29T18:16:20.821+0100] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: spark_airflow_project.python_job manual__2022-12-29T17:16:13.051785+00:00 [queued]>
[2022-12-29T18:16:20.821+0100] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-29T18:16:20.834+0100] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2022-12-29T18:16:20.834+0100] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-29T18:16:20.889+0100] {taskinstance.py:1304} INFO - Executing <Task(SparkSubmitOperator): python_job> on 2022-12-29 17:16:13.051785+00:00
[2022-12-29T18:16:20.895+0100] {standard_task_runner.py:55} INFO - Started process 5483 to run task
[2022-12-29T18:16:20.916+0100] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'spark_airflow_project', 'python_job', 'manual__2022-12-29T17:16:13.051785+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/spark_airflow.py', '--cfg-path', '/tmp/tmppknmb6n4']
[2022-12-29T18:16:20.920+0100] {standard_task_runner.py:83} INFO - Job 23: Subtask python_job
[2022-12-29T18:16:21.098+0100] {task_command.py:389} INFO - Running <TaskInstance: spark_airflow_project.python_job manual__2022-12-29T17:16:13.051785+00:00 [running]> on host momo-VirtualBox
[2022-12-29T18:16:21.235+0100] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Airflow
AIRFLOW_CTX_DAG_ID=spark_airflow_project
AIRFLOW_CTX_TASK_ID=python_job
AIRFLOW_CTX_EXECUTION_DATE=2022-12-29T17:16:13.051785+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-29T17:16:13.051785+00:00
[2022-12-29T18:16:21.257+0100] {base.py:73} INFO - Using connection ID 'spark_default' for task execution.
[2022-12-29T18:16:21.266+0100] {spark_submit.py:344} INFO - Spark-Submit cmd: spark-submit --master spark://momo-VirtualBox:7077 --name arrow-spark --queue root.default /home/momo/Bureau/spark_d2.py
[2022-12-29T18:16:28.445+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:28 WARN Utils: Your hostname, momo-VirtualBox resolves to a loopback address: 127.0.1.1; using 10.0.2.15 instead (on interface enp0s3)
[2022-12-29T18:16:28.450+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:28 WARN Utils: Set SPARK_LOCAL_IP if you need to bind to another address
[2022-12-29T18:16:31.580+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:31 INFO SparkContext: Running Spark version 3.3.1
[2022-12-29T18:16:31.964+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:31 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2022-12-29T18:16:32.443+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:32 INFO ResourceUtils: ==============================================================
[2022-12-29T18:16:32.445+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:32 INFO ResourceUtils: No custom resources configured for spark.driver.
[2022-12-29T18:16:32.458+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:32 INFO ResourceUtils: ==============================================================
[2022-12-29T18:16:32.461+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:32 INFO SparkContext: Submitted application: arrow-spark
[2022-12-29T18:16:32.598+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:32 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2022-12-29T18:16:32.658+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:32 INFO ResourceProfile: Limiting resource is cpu
[2022-12-29T18:16:32.660+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:32 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2022-12-29T18:16:32.819+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:32 INFO SecurityManager: Changing view acls to: momo
[2022-12-29T18:16:32.825+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:32 INFO SecurityManager: Changing modify acls to: momo
[2022-12-29T18:16:32.828+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:32 INFO SecurityManager: Changing view acls groups to:
[2022-12-29T18:16:32.838+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:32 INFO SecurityManager: Changing modify acls groups to:
[2022-12-29T18:16:32.840+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:32 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users  with view permissions: Set(momo); groups with view permissions: Set(); users  with modify permissions: Set(momo); groups with modify permissions: Set()
[2022-12-29T18:16:34.426+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:34 INFO Utils: Successfully started service 'sparkDriver' on port 43803.
[2022-12-29T18:16:34.573+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:34 INFO SparkEnv: Registering MapOutputTracker
[2022-12-29T18:16:34.762+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:34 INFO SparkEnv: Registering BlockManagerMaster
[2022-12-29T18:16:34.859+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:34 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2022-12-29T18:16:34.861+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:34 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2022-12-29T18:16:34.894+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:34 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2022-12-29T18:16:34.982+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:34 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-15276737-4484-4e18-abd1-fed19f77a636
[2022-12-29T18:16:35.080+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:35 INFO MemoryStore: MemoryStore started with capacity 413.9 MiB
[2022-12-29T18:16:35.203+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:35 INFO SparkEnv: Registering OutputCommitCoordinator
[2022-12-29T18:16:36.292+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:36 WARN Utils: Service 'SparkUI' could not bind on port 4040. Attempting port 4041.
[2022-12-29T18:16:36.331+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:36 INFO Utils: Successfully started service 'SparkUI' on port 4041.
[2022-12-29T18:16:37.257+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:37 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://momo-VirtualBox:7077...
[2022-12-29T18:16:37.564+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:37 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master momo-VirtualBox:7077
[2022-12-29T18:16:37.565+0100] {spark_submit.py:495} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2022-12-29T18:16:37.565+0100] {spark_submit.py:495} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2022-12-29T18:16:37.565+0100] {spark_submit.py:495} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2022-12-29T18:16:37.566+0100] {spark_submit.py:495} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:102)
[2022-12-29T18:16:37.566+0100] {spark_submit.py:495} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:110)
[2022-12-29T18:16:37.566+0100] {spark_submit.py:495} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2022-12-29T18:16:37.566+0100] {spark_submit.py:495} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2022-12-29T18:16:37.566+0100] {spark_submit.py:495} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2022-12-29T18:16:37.567+0100] {spark_submit.py:495} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2022-12-29T18:16:37.567+0100] {spark_submit.py:495} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2022-12-29T18:16:37.567+0100] {spark_submit.py:495} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2022-12-29T18:16:37.572+0100] {spark_submit.py:495} INFO - Caused by: java.io.IOException: Failed to connect to momo-VirtualBox/127.0.1.1:7077
[2022-12-29T18:16:37.572+0100] {spark_submit.py:495} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2022-12-29T18:16:37.572+0100] {spark_submit.py:495} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2022-12-29T18:16:37.573+0100] {spark_submit.py:495} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2022-12-29T18:16:37.573+0100] {spark_submit.py:495} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2022-12-29T18:16:37.573+0100] {spark_submit.py:495} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2022-12-29T18:16:37.576+0100] {spark_submit.py:495} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2022-12-29T18:16:37.576+0100] {spark_submit.py:495} INFO - ... 4 more
[2022-12-29T18:16:37.576+0100] {spark_submit.py:495} INFO - Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connexion refusée: momo-VirtualBox/127.0.1.1:7077
[2022-12-29T18:16:37.577+0100] {spark_submit.py:495} INFO - Caused by: java.net.ConnectException: Connexion refusée
[2022-12-29T18:16:37.577+0100] {spark_submit.py:495} INFO - at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
[2022-12-29T18:16:37.577+0100] {spark_submit.py:495} INFO - at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:777)
[2022-12-29T18:16:37.577+0100] {spark_submit.py:495} INFO - at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:330)
[2022-12-29T18:16:37.578+0100] {spark_submit.py:495} INFO - at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:334)
[2022-12-29T18:16:37.578+0100] {spark_submit.py:495} INFO - at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:710)
[2022-12-29T18:16:37.578+0100] {spark_submit.py:495} INFO - at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:658)
[2022-12-29T18:16:37.578+0100] {spark_submit.py:495} INFO - at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:584)
[2022-12-29T18:16:37.578+0100] {spark_submit.py:495} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:496)
[2022-12-29T18:16:37.578+0100] {spark_submit.py:495} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2022-12-29T18:16:37.578+0100] {spark_submit.py:495} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2022-12-29T18:16:37.579+0100] {spark_submit.py:495} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2022-12-29T18:16:37.579+0100] {spark_submit.py:495} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2022-12-29T18:16:57.241+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:57 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://momo-VirtualBox:7077...
[2022-12-29T18:16:57.283+0100] {spark_submit.py:495} INFO - 22/12/29 18:16:57 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master momo-VirtualBox:7077
[2022-12-29T18:16:57.284+0100] {spark_submit.py:495} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2022-12-29T18:16:57.284+0100] {spark_submit.py:495} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2022-12-29T18:16:57.285+0100] {spark_submit.py:495} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2022-12-29T18:16:57.289+0100] {spark_submit.py:495} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:102)
[2022-12-29T18:16:57.296+0100] {spark_submit.py:495} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:110)
[2022-12-29T18:16:57.298+0100] {spark_submit.py:495} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2022-12-29T18:16:57.298+0100] {spark_submit.py:495} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2022-12-29T18:16:57.299+0100] {spark_submit.py:495} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2022-12-29T18:16:57.299+0100] {spark_submit.py:495} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2022-12-29T18:16:57.299+0100] {spark_submit.py:495} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2022-12-29T18:16:57.300+0100] {spark_submit.py:495} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2022-12-29T18:16:57.300+0100] {spark_submit.py:495} INFO - Caused by: java.io.IOException: Failed to connect to momo-VirtualBox/127.0.1.1:7077
[2022-12-29T18:16:57.301+0100] {spark_submit.py:495} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2022-12-29T18:16:57.306+0100] {spark_submit.py:495} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2022-12-29T18:16:57.307+0100] {spark_submit.py:495} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2022-12-29T18:16:57.307+0100] {spark_submit.py:495} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2022-12-29T18:16:57.308+0100] {spark_submit.py:495} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2022-12-29T18:16:57.308+0100] {spark_submit.py:495} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2022-12-29T18:16:57.308+0100] {spark_submit.py:495} INFO - ... 4 more
[2022-12-29T18:16:57.309+0100] {spark_submit.py:495} INFO - Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connexion refusée: momo-VirtualBox/127.0.1.1:7077
[2022-12-29T18:16:57.309+0100] {spark_submit.py:495} INFO - Caused by: java.net.ConnectException: Connexion refusée
[2022-12-29T18:16:57.314+0100] {spark_submit.py:495} INFO - at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
[2022-12-29T18:16:57.314+0100] {spark_submit.py:495} INFO - at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:777)
[2022-12-29T18:16:57.315+0100] {spark_submit.py:495} INFO - at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:330)
[2022-12-29T18:16:57.315+0100] {spark_submit.py:495} INFO - at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:334)
[2022-12-29T18:16:57.315+0100] {spark_submit.py:495} INFO - at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:710)
[2022-12-29T18:16:57.316+0100] {spark_submit.py:495} INFO - at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:658)
[2022-12-29T18:16:57.316+0100] {spark_submit.py:495} INFO - at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:584)
[2022-12-29T18:16:57.317+0100] {spark_submit.py:495} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:496)
[2022-12-29T18:16:57.317+0100] {spark_submit.py:495} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2022-12-29T18:16:57.323+0100] {spark_submit.py:495} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2022-12-29T18:16:57.323+0100] {spark_submit.py:495} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2022-12-29T18:16:57.324+0100] {spark_submit.py:495} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2022-12-29T18:17:17.251+0100] {spark_submit.py:495} INFO - 22/12/29 18:17:17 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://momo-VirtualBox:7077...
[2022-12-29T18:17:17.279+0100] {spark_submit.py:495} INFO - 22/12/29 18:17:17 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master momo-VirtualBox:7077
[2022-12-29T18:17:17.279+0100] {spark_submit.py:495} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2022-12-29T18:17:17.280+0100] {spark_submit.py:495} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2022-12-29T18:17:17.280+0100] {spark_submit.py:495} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2022-12-29T18:17:17.280+0100] {spark_submit.py:495} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:102)
[2022-12-29T18:17:17.280+0100] {spark_submit.py:495} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:110)
[2022-12-29T18:17:17.280+0100] {spark_submit.py:495} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2022-12-29T18:17:17.280+0100] {spark_submit.py:495} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2022-12-29T18:17:17.281+0100] {spark_submit.py:495} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2022-12-29T18:17:17.281+0100] {spark_submit.py:495} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2022-12-29T18:17:17.281+0100] {spark_submit.py:495} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2022-12-29T18:17:17.281+0100] {spark_submit.py:495} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2022-12-29T18:17:17.286+0100] {spark_submit.py:495} INFO - Caused by: java.io.IOException: Failed to connect to momo-VirtualBox/127.0.1.1:7077
[2022-12-29T18:17:17.286+0100] {spark_submit.py:495} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2022-12-29T18:17:17.286+0100] {spark_submit.py:495} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2022-12-29T18:17:17.286+0100] {spark_submit.py:495} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2022-12-29T18:17:17.287+0100] {spark_submit.py:495} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2022-12-29T18:17:17.287+0100] {spark_submit.py:495} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2022-12-29T18:17:17.287+0100] {spark_submit.py:495} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2022-12-29T18:17:17.287+0100] {spark_submit.py:495} INFO - ... 4 more
[2022-12-29T18:17:17.287+0100] {spark_submit.py:495} INFO - Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connexion refusée: momo-VirtualBox/127.0.1.1:7077
[2022-12-29T18:17:17.287+0100] {spark_submit.py:495} INFO - Caused by: java.net.ConnectException: Connexion refusée
[2022-12-29T18:17:17.288+0100] {spark_submit.py:495} INFO - at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
[2022-12-29T18:17:17.288+0100] {spark_submit.py:495} INFO - at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:777)
[2022-12-29T18:17:17.288+0100] {spark_submit.py:495} INFO - at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:330)
[2022-12-29T18:17:17.288+0100] {spark_submit.py:495} INFO - at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:334)
[2022-12-29T18:17:17.288+0100] {spark_submit.py:495} INFO - at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:710)
[2022-12-29T18:17:17.288+0100] {spark_submit.py:495} INFO - at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:658)
[2022-12-29T18:17:17.289+0100] {spark_submit.py:495} INFO - at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:584)
[2022-12-29T18:17:17.289+0100] {spark_submit.py:495} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:496)
[2022-12-29T18:17:17.289+0100] {spark_submit.py:495} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2022-12-29T18:17:17.298+0100] {spark_submit.py:495} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2022-12-29T18:17:17.298+0100] {spark_submit.py:495} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2022-12-29T18:17:17.298+0100] {spark_submit.py:495} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2022-12-29T18:17:37.251+0100] {spark_submit.py:495} INFO - 22/12/29 18:17:37 ERROR StandaloneSchedulerBackend: Application has been killed. Reason: All masters are unresponsive! Giving up.
[2022-12-29T18:17:37.261+0100] {spark_submit.py:495} INFO - 22/12/29 18:17:37 WARN StandaloneSchedulerBackend: Application ID is not initialized yet.
[2022-12-29T18:17:37.309+0100] {spark_submit.py:495} INFO - 22/12/29 18:17:37 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 40275.
[2022-12-29T18:17:37.311+0100] {spark_submit.py:495} INFO - 22/12/29 18:17:37 INFO NettyBlockTransferService: Server created on 10.0.2.15:40275
[2022-12-29T18:17:37.326+0100] {spark_submit.py:495} INFO - 22/12/29 18:17:37 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2022-12-29T18:17:37.379+0100] {spark_submit.py:495} INFO - 22/12/29 18:17:37 INFO SparkUI: Stopped Spark web UI at http://10.0.2.15:4041
[2022-12-29T18:17:37.389+0100] {spark_submit.py:495} INFO - 22/12/29 18:17:37 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, 10.0.2.15, 40275, None)
[2022-12-29T18:17:37.435+0100] {spark_submit.py:495} INFO - 22/12/29 18:17:37 INFO StandaloneSchedulerBackend: Shutting down all executors
[2022-12-29T18:17:37.465+0100] {spark_submit.py:495} INFO - 22/12/29 18:17:37 INFO BlockManagerMasterEndpoint: Registering block manager 10.0.2.15:40275 with 413.9 MiB RAM, BlockManagerId(driver, 10.0.2.15, 40275, None)
[2022-12-29T18:17:37.472+0100] {spark_submit.py:495} INFO - 22/12/29 18:17:37 INFO CoarseGrainedSchedulerBackend$DriverEndpoint: Asking each executor to shut down
[2022-12-29T18:17:37.499+0100] {spark_submit.py:495} INFO - 22/12/29 18:17:37 WARN StandaloneAppClient$ClientEndpoint: Drop UnregisterApplication(null) because has not yet connected to master
[2022-12-29T18:17:37.505+0100] {spark_submit.py:495} INFO - 22/12/29 18:17:37 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, 10.0.2.15, 40275, None)
[2022-12-29T18:17:37.516+0100] {spark_submit.py:495} INFO - 22/12/29 18:17:37 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, 10.0.2.15, 40275, None)
[2022-12-29T18:17:37.622+0100] {spark_submit.py:495} INFO - 22/12/29 18:17:37 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2022-12-29T18:17:37.731+0100] {spark_submit.py:495} INFO - 22/12/29 18:17:37 INFO MemoryStore: MemoryStore cleared
[2022-12-29T18:17:37.734+0100] {spark_submit.py:495} INFO - 22/12/29 18:17:37 INFO BlockManager: BlockManager stopped
[2022-12-29T18:17:37.806+0100] {spark_submit.py:495} INFO - 22/12/29 18:17:37 INFO BlockManagerMaster: BlockManagerMaster stopped
[2022-12-29T18:17:37.848+0100] {spark_submit.py:495} INFO - 22/12/29 18:17:37 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2022-12-29T18:17:37.931+0100] {spark_submit.py:495} INFO - 22/12/29 18:17:37 INFO SparkContext: Successfully stopped SparkContext
[2022-12-29T18:17:38.730+0100] {spark_submit.py:495} INFO - 22/12/29 18:17:38 ERROR AsyncEventQueue: Listener AppStatusListener threw an exception
[2022-12-29T18:17:38.730+0100] {spark_submit.py:495} INFO - java.lang.NullPointerException
[2022-12-29T18:17:38.730+0100] {spark_submit.py:495} INFO - at org.apache.spark.status.AppStatusListener.onApplicationEnd(AppStatusListener.scala:192)
[2022-12-29T18:17:38.731+0100] {spark_submit.py:495} INFO - at org.apache.spark.scheduler.SparkListenerBus.doPostEvent(SparkListenerBus.scala:57)
[2022-12-29T18:17:38.731+0100] {spark_submit.py:495} INFO - at org.apache.spark.scheduler.SparkListenerBus.doPostEvent$(SparkListenerBus.scala:28)
[2022-12-29T18:17:38.731+0100] {spark_submit.py:495} INFO - at org.apache.spark.scheduler.AsyncEventQueue.doPostEvent(AsyncEventQueue.scala:37)
[2022-12-29T18:17:38.731+0100] {spark_submit.py:495} INFO - at org.apache.spark.scheduler.AsyncEventQueue.doPostEvent(AsyncEventQueue.scala:37)
[2022-12-29T18:17:38.731+0100] {spark_submit.py:495} INFO - at org.apache.spark.util.ListenerBus.postToAll(ListenerBus.scala:117)
[2022-12-29T18:17:38.731+0100] {spark_submit.py:495} INFO - at org.apache.spark.util.ListenerBus.postToAll$(ListenerBus.scala:101)
[2022-12-29T18:17:38.731+0100] {spark_submit.py:495} INFO - at org.apache.spark.scheduler.AsyncEventQueue.super$postToAll(AsyncEventQueue.scala:105)
[2022-12-29T18:17:38.731+0100] {spark_submit.py:495} INFO - at org.apache.spark.scheduler.AsyncEventQueue.$anonfun$dispatch$1(AsyncEventQueue.scala:105)
[2022-12-29T18:17:38.732+0100] {spark_submit.py:495} INFO - at scala.runtime.java8.JFunction0$mcJ$sp.apply(JFunction0$mcJ$sp.java:23)
[2022-12-29T18:17:38.732+0100] {spark_submit.py:495} INFO - at scala.util.DynamicVariable.withValue(DynamicVariable.scala:62)
[2022-12-29T18:17:38.732+0100] {spark_submit.py:495} INFO - at org.apache.spark.scheduler.AsyncEventQueue.org$apache$spark$scheduler$AsyncEventQueue$$dispatch(AsyncEventQueue.scala:100)
[2022-12-29T18:17:38.733+0100] {spark_submit.py:495} INFO - at org.apache.spark.scheduler.AsyncEventQueue$$anon$2.$anonfun$run$1(AsyncEventQueue.scala:96)
[2022-12-29T18:17:38.733+0100] {spark_submit.py:495} INFO - at org.apache.spark.util.Utils$.tryOrStopSparkContext(Utils.scala:1446)
[2022-12-29T18:17:38.733+0100] {spark_submit.py:495} INFO - at org.apache.spark.scheduler.AsyncEventQueue$$anon$2.run(AsyncEventQueue.scala:96)
[2022-12-29T18:17:39.015+0100] {spark_submit.py:495} INFO - 22/12/29 18:17:39 ERROR SparkContext: Error initializing SparkContext.
[2022-12-29T18:17:39.016+0100] {spark_submit.py:495} INFO - java.lang.IllegalArgumentException: requirement failed: Can only call getServletHandlers on a running MetricsSystem
[2022-12-29T18:17:39.016+0100] {spark_submit.py:495} INFO - at scala.Predef$.require(Predef.scala:281)
[2022-12-29T18:17:39.016+0100] {spark_submit.py:495} INFO - at org.apache.spark.metrics.MetricsSystem.getServletHandlers(MetricsSystem.scala:89)
[2022-12-29T18:17:39.017+0100] {spark_submit.py:495} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:650)
[2022-12-29T18:17:39.017+0100] {spark_submit.py:495} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2022-12-29T18:17:39.017+0100] {spark_submit.py:495} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2022-12-29T18:17:39.017+0100] {spark_submit.py:495} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2022-12-29T18:17:39.017+0100] {spark_submit.py:495} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2022-12-29T18:17:39.018+0100] {spark_submit.py:495} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2022-12-29T18:17:39.018+0100] {spark_submit.py:495} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2022-12-29T18:17:39.023+0100] {spark_submit.py:495} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
[2022-12-29T18:17:39.023+0100] {spark_submit.py:495} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2022-12-29T18:17:39.023+0100] {spark_submit.py:495} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2022-12-29T18:17:39.023+0100] {spark_submit.py:495} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2022-12-29T18:17:39.023+0100] {spark_submit.py:495} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2022-12-29T18:17:39.023+0100] {spark_submit.py:495} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2022-12-29T18:17:39.024+0100] {spark_submit.py:495} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2022-12-29T18:17:39.036+0100] {spark_submit.py:495} INFO - 22/12/29 18:17:39 INFO SparkContext: SparkContext already stopped.
[2022-12-29T18:17:39.382+0100] {spark_submit.py:495} INFO - Traceback (most recent call last):
[2022-12-29T18:17:39.383+0100] {spark_submit.py:495} INFO - File "/home/momo/Bureau/spark_d2.py", line 9, in <module>
[2022-12-29T18:17:39.383+0100] {spark_submit.py:495} INFO - sc= SparkContext()
[2022-12-29T18:17:39.383+0100] {spark_submit.py:495} INFO - File "/home/momo/.local/lib/python3.8/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 197, in __init__
[2022-12-29T18:17:39.384+0100] {spark_submit.py:495} INFO - File "/home/momo/.local/lib/python3.8/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 282, in _do_init
[2022-12-29T18:17:39.384+0100] {spark_submit.py:495} INFO - File "/home/momo/.local/lib/python3.8/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 402, in _initialize_context
[2022-12-29T18:17:39.384+0100] {spark_submit.py:495} INFO - File "/home/momo/.local/lib/python3.8/site-packages/pyspark/python/lib/py4j-0.10.9.5-src.zip/py4j/java_gateway.py", line 1585, in __call__
[2022-12-29T18:17:39.384+0100] {spark_submit.py:495} INFO - File "/home/momo/.local/lib/python3.8/site-packages/pyspark/python/lib/py4j-0.10.9.5-src.zip/py4j/protocol.py", line 326, in get_return_value
[2022-12-29T18:17:39.390+0100] {spark_submit.py:495} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling None.org.apache.spark.api.java.JavaSparkContext.
[2022-12-29T18:17:39.391+0100] {spark_submit.py:495} INFO - : java.lang.IllegalArgumentException: requirement failed: Can only call getServletHandlers on a running MetricsSystem
[2022-12-29T18:17:39.391+0100] {spark_submit.py:495} INFO - at scala.Predef$.require(Predef.scala:281)
[2022-12-29T18:17:39.391+0100] {spark_submit.py:495} INFO - at org.apache.spark.metrics.MetricsSystem.getServletHandlers(MetricsSystem.scala:89)
[2022-12-29T18:17:39.391+0100] {spark_submit.py:495} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:650)
[2022-12-29T18:17:39.391+0100] {spark_submit.py:495} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2022-12-29T18:17:39.391+0100] {spark_submit.py:495} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2022-12-29T18:17:39.391+0100] {spark_submit.py:495} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2022-12-29T18:17:39.391+0100] {spark_submit.py:495} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2022-12-29T18:17:39.391+0100] {spark_submit.py:495} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2022-12-29T18:17:39.391+0100] {spark_submit.py:495} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2022-12-29T18:17:39.392+0100] {spark_submit.py:495} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
[2022-12-29T18:17:39.392+0100] {spark_submit.py:495} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2022-12-29T18:17:39.392+0100] {spark_submit.py:495} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2022-12-29T18:17:39.392+0100] {spark_submit.py:495} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2022-12-29T18:17:39.392+0100] {spark_submit.py:495} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2022-12-29T18:17:39.392+0100] {spark_submit.py:495} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2022-12-29T18:17:39.392+0100] {spark_submit.py:495} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2022-12-29T18:17:39.397+0100] {spark_submit.py:495} INFO - 
[2022-12-29T18:17:39.533+0100] {spark_submit.py:495} INFO - 22/12/29 18:17:39 INFO ShutdownHookManager: Shutdown hook called
[2022-12-29T18:17:39.538+0100] {spark_submit.py:495} INFO - 22/12/29 18:17:39 INFO ShutdownHookManager: Deleting directory /tmp/spark-b465c7e0-f6c0-4f11-a441-88b5226fb006
[2022-12-29T18:17:39.546+0100] {spark_submit.py:495} INFO - 22/12/29 18:17:39 INFO ShutdownHookManager: Deleting directory /tmp/spark-68cad760-4a8f-47a6-be46-a61b88efc6dc
[2022-12-29T18:17:39.664+0100] {taskinstance.py:1772} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/momo/.local/lib/python3.8/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 157, in execute
    self._hook.submit(self._application)
  File "/home/momo/.local/lib/python3.8/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 426, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://momo-VirtualBox:7077 --name arrow-spark --queue root.default /home/momo/Bureau/spark_d2.py. Error code is: 1.
[2022-12-29T18:17:39.671+0100] {taskinstance.py:1322} INFO - Marking task as FAILED. dag_id=spark_airflow_project, task_id=python_job, execution_date=20221229T171613, start_date=20221229T171620, end_date=20221229T171739
[2022-12-29T18:17:39.694+0100] {standard_task_runner.py:100} ERROR - Failed to execute job 23 for task python_job (Cannot execute: spark-submit --master spark://momo-VirtualBox:7077 --name arrow-spark --queue root.default /home/momo/Bureau/spark_d2.py. Error code is: 1.; 5483)
[2022-12-29T18:17:39.730+0100] {local_task_job.py:159} INFO - Task exited with return code 1
[2022-12-29T18:17:39.763+0100] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
